docs/en/status-communities/about-view-only-channels.md

```diff
@@ -0,0 +1,57 @@
+---
+id: 662
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# About view-only channels
+
+!!! note ""
+    Currently, you can only configure view-only channels using Status desktop. [:octicons-desktop-download-16: Download Status desktop][status-web-download].
+
+[Channels][channels-your-quick-start-guide] represent spaces for conversation, discussion and information sharing. By default, all community members with access to a channel can participate in the conversation, reading and posting messages.
+
+As a community [owner or admin][permissions-by-role-in-status-communities], you can set a channel as view-only. For specific instructions, check out [Set up a view-only channel][set-up-a-view-only-channel]. 
+
+A read-only channel aims to facilitate clear, uninterrupted communication for specific purposes. While members can't respond directly in these channels, they are crucial in keeping your community informed and engaged.
```

Comment:
'aims to facilitate' -> 'helps maintain'? Again, I'm trying to simplify sentences to make them easier to scan.